mk/envoy.mk

```diff
@@ -0,0 +1,26 @@
+ENVOY_TAG ?= v1.18.4
+ENVOY_COMMIT_HASH ?=
+
+SOURCE_DIR ?= ${TMPDIR}envoy-sources
+ifndef TMPDIR
+	SOURCE_DIR ?= /tmp/envoy-sources
+endif
+
+BUILD_ENVOY_FROM_SOURCES ?= false
+
+.PHONY: build/envoy
+build/envoy:
```

Comment:
Add a help string to this one?

Reply:
Sorry, I wasn't very clear here. What I meant was a `##` comment so that this target shows up in the output of `make help`